app/views/pages/developer-guide/forms/validation.liquid

```diff
@@ -74,6 +74,17 @@ password_confirmation:
     virtual: true
 ```
 
+### email
+
+Checks if a provided value is a valid email address.
+
+```yaml
+author_email:
+  validation:
+    email: true
+    presence: true
```

Comment:
Is presence required to demonstrate it?

Reply:
no, it's not - we can remove it, however, that's the most common scenario

Reply:
i would remove it then to not confuse reader

Reply:
```suggestion
```